mitodl/mit-open
# adding migrations for copy update

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/4896



### Description (What does it do?)
Adds a data_fixtures migration to make the copy updates as described in  https://github.com/mitodl/hq/issues/4896


### How can this be tested?
1. checkout this branch
2. run data_fixture migrations via ```RUN_DATA_MIGRATIONS=true python  manage.py migrate```
3. verify the copy has been updated on unit pages for ocw, mitpe and mitx
